src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1280,6 +1280,40 @@ public static void ContainsNoMatch_StringComparison()
             }
         }
 
+        [Fact]
+        public static void Contains_Match_Char() // Adapted from IndexOf_Match_SingleLetter
```

Comment:
```text
15:21:33 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Tests\System\StringTests.cs(1287,38): error CS7036: There is no argument given that corresponds to the required formal parameter 'comparisonType' of 'MemoryExtensions.Contains(ReadOnlySpan<char>, ReadOnlySpan<char>, StringComparison)' [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Memory\tests\System.Memory.Tests.csproj]
15:21:33 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Tests\System\StringTests.cs(1308,34): error CS7036: There is no argument given that corresponds to the required formal parameter 'comparisonType' of 'MemoryExtensions.Contains(ReadOnlySpan<char>, ReadOnlySpan<char>, StringComparison)' [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Memory\tests\System.Memory.Tests.csproj]
15:21:33 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\tests\Tests\System\StringTests.cs(1311,33): error CS7036: There is no argument given that corresponds to the required formal parameter 'comparisonType' of 'MemoryExtensions.Contains(ReadOnlySpan<char>, ReadOnlySpan<char>, StringComparison)' [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Memory\tests\System.Memory.Tests.csproj]
```

Since the new API is for .NET Core only, I think we should add this test here instead: https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System/StringTests.netcoreapp.cs

Reply:
Fixed